### PR TITLE
5_Added tests for new methods in HdfsUtils

### PR DESCRIPTION
**Pull request by pdtran3k6**
Majority of this PR is from https://github.com/linkedin/TonY/pull/104. To view the changes I made from https://github.com/linkedin/TonY/pull/104, scroll down and click on the latest commit.

### CODE REVIEW

File: tony-history-server/test/utils/TestHdfsUtils.java [hunk 1]
```diff
@@ -27,6 +28,71 @@ public static void setup() {
...
+  @Test
+  public void testDeleteMultiDir() throws IOException {
+    Path deleteDirs = new Path("./test/resources/deleteDirs");
+    for (int i = 0; i < 5; i++) {
+      fs.mkdirs(new Path(deleteDirs, "dir" + i));
+    }
+    FileStatus[] statuses = fs.listStatus(deleteDirs);
+
+    HdfsUtils.deleteMultiDir(fs, statuses);
+    assertEquals(0, fs.listStatus(deleteDirs).length);
```
Comment (erwa) [inline tony-history-server/test/utils/TestHdfsUtils.java]: Before the `deleteMultiDir` line, I would suggest adding an `assertEquals(5, fs.listStatus(deleteDirs).length);` check just as a sanity that 5 directories were created successfully before we deleted them.

Comment (pdtran3k6) [reply tony-history-server/test/utils/TestHdfsUtils.java]: Fixed.

**Comment by erwa** [comment]
CI build failed due to:
```
> Task :tony-core:compileJava
/home/travis/build/linkedin/TonY/tony-core/src/main/java/com/linkedin/tony/TonyConfigurationKeys.java:46: error: variable TONY_HISTORY_INTERMEDIATE is already defined in class TonyConfigurationKeys
  public static final String TONY_HISTORY_INTERMEDIATE = TONY_PREFIX + "history.intermediate";
                             ^
/home/travis/build/linkedin/TonY/tony-core/src/main/java/com/linkedin/tony/TonyConfigurationKeys.java:47: error: variable DEFAULT_TONY_HISTORY_INTERMEDIATE is already defined in class TonyConfigurationKeys
  public static final String DEFAULT_TONY_HISTORY_INTERMEDIATE = DEFAULT_TONY_HISTORY_LOCATION + "/intermediate";
                             ^
/home/travis/build/linkedin/TonY/tony-core/src/main/java/com/linkedin/tony/TonyConfigurationKeys.java:49: error: variable TONY_HISTORY_FINISHED is already defined in class TonyConfigurationKeys
  public static final String TONY_HISTORY_FINISHED = TONY_PREFIX + "history.finished";
                             ^
/home/travis/build/linkedin/TonY/tony-core/src/main/java/com/linkedin/tony/TonyConfigurationKeys.java:50: error: variable DEFAULT_TONY_HISTORY_FINISHED is already defined in class TonyConfigurationKeys
  public static final String DEFAULT_TONY_HISTORY_FINISHED = DEFAULT_TONY_HISTORY_LOCATION + "/finished";
```

**Comment by erwa** [comment]
GitHub says there are conflicts that must be resolved. Can you resolve them and update this PR?

**Pull request merged**